lib/cute_pets/pet_fetcher.rb

```diff
@@ -21,6 +21,9 @@ def get_petfinder_pet()
 
     if response.kind_of? Net::HTTPSuccess
       json = JSON.parse(response.body)
+      if json['petfinder']['header']['status']['message']['$t'] == 'shelter opt-out'
+        raise 'The chosen shelter opted out of being accesible via the API'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.